src/common/pagination.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2020 BlockDev AG
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { validateMultiple } from '../fmt/validation'
+
+export interface Pagination {
+  totalItems: number
+  totalPages: number
+  currentPage: number
+  previousPage: number
+  nextPage: number
+}
```

Comment:
```suggestion
export interface Pageable {
  page: number
  pageSize: number
  totalItems: number
  totalPages: number
}
```

Reply:
Previous page is always `max(page-1, 1)`, next page is always `min(page+1, totalPages)`, thus it's not necessary to return from the API. But it is important to know the page size.